spec/fixtures/json/cqm_measures/CMS123v7/CMS123v7.json

```diff
@@ -7099,10 +7100,12 @@
           "statement_name": "Initial Population",
           "statement_references": [
             {
+              "hqmf_id": null,
```

Comment:
Are these hqmf_id changes from a previous commit?

Reply:
I didn't touch the hqmf_id stuff in this update. The only thing that changed was adding `is_top_level` to the new fixtures